db/migrations/60_merge_users_items_into_groups_attempts.sql

```diff
@@ -23,15 +27,43 @@ SELECT
     `users_items`.`autonomy`,`users_items`.`started_at`,`users_items`.`validated_at`,
     `users_items`.`finished_at`,`users_items`.`latest_activity_at`,`users_items`.`thread_started_at`,
     `users_items`.`best_answer_at`,`users_items`.`latest_answer_at`,`users_items`.`latest_hint_at`,
-    `users_items`.`ranked`,`users_items`.`all_lang_prog`
+    `users_items`.`ranked`,`users_items`.`all_lang_prog`,
+    IF(`users`.`self_group_id` = `groups_to_insert`.`id`, 'done', 'todo')
```

Comment:
We recompute stats for every item having at least one descendant marked as 'todo'. So should work fine (I believe).

Reply:
I don't think it works as you would have: 
- one leaf of single-user participation  -> copied -> marked as done: ok
- the parent contest done alone -> copied -> done: ok
- the parent chapter -> copied -> done: this node may be made of 2 contests, one the user has done alone (above) and one the user has done in team. In groups_attempts related to the self_group_id, you should only have results of the user alone, if you recompute, it will be correct but what you have copied is not.
Right? 🤔

Reply:
right. Fixed